improver/calibration/rainforest_calibration.py

```diff
@@ -450,12 +446,13 @@ def _evaluate_probabilities(
                 forecast_bool = forecast_data + threshold >= lower_bound_in_fcst_units
                 if np.any(forecast_bool):
                     input_subset = input_data[forecast_bool]
-                    if model_input_converter:
-                        input_subset = model_input_converter(input_subset)
+                    if self.model_input_converter:
```

Comment:
I think the `if` statement is not necessary here, because `self.model_input_converter` is always defined. (This is my mistake actually, when I wrote this code the first time.)

Reply:
I've done this in 4829778.